.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Feature request
+about: Suggest an idea for this project.
+labels: 'type: feature request'
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
Needs a "or feature" section ? Or could you upgrade this line such as "Is your feature request related to a problem or a new feature? Please describe"

Reply:
This is the GitHub default. Note that this is not a template for *both* bug reports and feature requests. It's specifically for feature requests. There's another template (in the  same PR) for bug reports.